src/components/common/Input.jsx

```diff
@@ -0,0 +1,25 @@
+import { useState } from 'react';
+import styles from './Input.module.css';
+
+const Input = ({ onChange, ...otherProps }) => {
```

Comment:
why do we need this otherProps, what for?

Reply:
We pass props like placeholder and type into the <input /> without changes.